sdk/utils.ts

```diff
@@ -152,4 +152,26 @@ export const getAlerts = async (query: AlertQueryOptions): Promise<AlertsRespons
   if(response.data && response.data.errors) throw Error(response.data.errors)
 
   return response.data.data.alerts
+}
+
+export const fetchJwtToken = async (claims: {}): Promise<{token: string} | null> => {
```

Comment:
If the bot sends a `exp` claim the scanner will use it as an expiration time.  As an SDK interface, I'm thinking it might make sense to have a first class optional `expiresAt` as a parameter.  

Something like this:
```
export const fetchJwtToken = async (claims: {}, expiresAt: Date | undefined): Promise<{token: string} | null> => {
  if( expiresAt !== undefined ) {
    claims["exp"] = Math.floor(expiresAt.getTime()/1000);
  }
  ...
}
```
What do you think?   The other option is to change nothing and expect bots to pass `exp` as a claim with seconds, I just wonder if it would work well.  Either way, but wanted to mention this as an option.

Reply:
That makes sense. The sdk should support 100% the functionality offered by a scanner.